src/pages/components/text-input/usage.mdx

```diff
@@ -524,6 +526,22 @@ has been met.
 </Column>
 </Row>
 
+### Word counter
+
+A word counter can be added to text area to indicate both the number of words
+being entered and the total number of words allowed. Once the max number of
```

Comment:
>  Once the max number of characters is reached the text area should prevent the user from entering any additional character and provide messaging to the user that a limit has been met.

Should "characters" be changed to be called "words" here under the new Word counter section? Technically it seems that you can keep typing additional characters as long as it is still part of the last word available in the counter. [Storybook example](https://react.carbondesignsystem.com/?path=/story/components-textarea--playground&args=maxCount:10;counterMode:word)